rosidl_typesupport_opensplice_cpp/include/rosidl_typesupport_opensplice_cpp/requester.hpp

```diff
@@ -137,10 +139,8 @@ class Requester
       estr = "DomainParticipant::create_topic: failed for response";
       goto fail;
     }
-
-    // Let OpenSplice do any needed encoding
     content_filtered_response_topic_ = participant_->create_contentfilteredtopic(
-      service_name_.c_str(), response_topic_,
+      content_filtered_topic_name.c_str(), response_topic_,
```

Comment:
This shouldn't be necessary. The purpose of using a `ContentFilteredTopic` is such that we reuse the same topic name, but let the filtering happen on the content (hence the query). DDS would inspect the fields of a message and if they match the query, the message will be routed to the appropriate client.